global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file is required for me to run `dotnet ef` as the ef CLI tool will not work on the latest version of the .Net SDK

(As discussed with Jan) We may need to change this again when we ensure that everyone is running the same `dotnet` SDK version